pages/housing.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+permalink: /housing
+title: Housing
+---
+
+_Designed by world-famous Spanish architect Josep Lluis Sera and completed in 1982, Next House sports architecture themed around sun and community with colorful living spaces, a bright dining hall, and a beautiful courtyard._
+
+Next House is a co-ed dorm with four floors to house over 350
+students. Each floor is divided into an east and west side,
+giving Next eight **wings**, each with their own distinct
+atmosphere and culture.
+
+![Next House layout]({{ site.url }}/static/assets/layout.png)
+
+Typically, students room with those of the same gender.
+However, **gender-inclusive housing (GIH)** is a policy that
+provides MIT students with the option to live in residences
+with whomever they choose, regardless of biological sex,
+gender, or gender identity. Any resident is welcome to
+participate in GIH. For more information about GIH, please
+visit the Housing website. If you do not opt in to GIH, you
+are guaranteed to live with someone of the same gender.
+
+## Rooms @ Next
+
+Next House has singles, doubles and triples. While rooms vary in shape and size, all of the rooms in Next House are large compared to some other undergraduate dorms. On average, singles are about 125 square feet, and doubles are about twice that size. The largest rooms in Next House are the solar singles and doubles, named for their floor-to-ceiling windows. Most freshmen will live in doubles and triples, although there are usually a few singles open in the Fall.
+
+![A typical double]({{ site.url }}/static/assets/double.jpg)
+
+![A typical single]({{ site.url }}/static/assets/single.jpg)
+
+Each resident will be provided with a bed, desk, bookcase, dresser, trash can, Ethernet connection, cable connection, and closet space in their room. Rooms on the river-side of the dorm have gorgeous views of the Charles River and Boston. Rooms on the opposite side have a great view of the athletic fields and Simmons Hall.
+
+## Wing Spaces @ Next
+
+![Wing lounge]({{ site.url }}/static/assets/lounge.jpg)
+
+Each wing has a large lounge with floor-to-ceiling windows overlooking the Charles River and Boston, a TV connected to MIT cable, large whiteboards, and counter space for cooking. In addition to the large lounge, each wing has smaller lounges, including bike lounges for bike storage, and mini-lounges for hanging out and studying.
+
+![Bathroom]({{ site.url }}/static/assets/bathroom.jpg)
```

Comment:
Hmm yeah, I'm not sure how to do inline images w/ jekyll, so maybe in the meantime we can get bigger images?